packages/react-components/react-breadcrumb/src/components/Breadcrumb/useBreadcrumb.ts

```diff
@@ -17,18 +18,28 @@ export const useBreadcrumb_unstable = (props: BreadcrumbProps, ref: React.Ref<HT
     dividerType = 'chevron',
     iconPosition = 'before',
     size = 'medium',
+    list,
     ...rest
   } = props;
+
+  const focusAttributes = useArrowNavigationGroup({
+    circular: true,
+    axis: 'horizontal',
+    memorizeCurrent: true,
+  });
+
   return {
     components: {
       root: 'nav',
       list: 'ol',
     },
     root: getNativeElementProps('nav', {
       ref,
+      'aria-label': 'breadcrumb' || props['aria-label'],
+      ...focusAttributes,
```

Comment:
I believe this should be conditional based on a prop -- I get why apps like Teams would want to arrow through a breadcrumb nav, but when this is used on websites each item should be in the tab order.

Reply:
When I asked designers if we need to add opportunity to turn it off they say that focus behavior should be turned on. Maybe it wound be good to add a prop with opportunity to turn it off, but by default it should be on. I'll double-check with the designers.

Reply:
I added opportunity to disable focus.